packages/lib-user/src/components/shared/TopProjects/TopProjects.js

```diff
@@ -27,12 +39,21 @@ CardsGrid.propTypes = {
 function CardsRow({ children }) {
   return (
     <Box
+      a11yTitle='Top Projects'
       as='ul'
       direction='row'
       gap='small'
-      pad={{ horizontal: 'xxsmall', bottom: 'xsmall', top: 'xxsmall' }}
       overflow={{ horizontal: 'auto' }}
-      style={{ listStyle: 'none' }}
+      pad={{
+        bottom: 'xsmall',
+        horizontal: 'xxsmall',
+        top: 'xxsmall'
+      }}
+      style={{
+        listStyle: 'none',
+        marginBlockEnd: 0,
+        marginBlockStart: 0,
+      }}
```

Comment:
Are you intentionally using a `style` object here instead of the `styled` function? I noticed StyledGridList has these CSS properties in a `styled` component.

Reply:
Please put it in a `styled` function and this PR is good to go! It looks nice run locally.

Reply:
All set in - [3ddf58f](https://github.com/zooniverse/front-end-monorepo/pull/6317/commits/3ddf58fcad1165635a6426883266289d62430b9e)